ai-catalog/samples/gemini-live-todo/src/main/java/com/android/ai/samples/geminilivetodo/ui/TodoScreen.kt

```diff
@@ -86,11 +91,21 @@ import kotlin.collections.reversed
 fun TodoScreen(viewModel: TodoScreenViewModel = hiltViewModel()) {
     val uiState by viewModel.uiState.collectAsStateWithLifecycle()
     var text by remember { mutableStateOf("") }
+    val context = LocalContext.current
```

Comment:
Optional but there is a library you could use which would be slightly less code and not use Context here

https://github.com/google/accompanist/tree/main/permissions